Ginger/GingerCoreCommon/Dictionaries/GingerTerminology.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+//namespace Amdocs.Ginger.Common.Dictionaries
+namespace GingerCore
+{
+    public class GingerTerminology
+    {
+        public static eTerminologyDicsType SET_TERMINOLOGY_TYPE { get; set; }
+
+        static List<KeyValuePair<string, string>> gingerTermDefaultList = new List<KeyValuePair<string, string>>()
+        {
+            new KeyValuePair<string, string>("BusinessFlow", "Business Flow Feature"),
+            new KeyValuePair<string, string>("BusinessFlows", "Business Flow Features"),
+            new KeyValuePair<string, string>("ActivitiesGroup", "Activities Group"),
```

Comment:
("BusinessFlow", "Business Flow Feature"), is wrong, should be ("BusinessFlow", "Business Flow"), also ("BusinessFlows", "Business Flow Features") -->> ("BusinessFlows", "Business Flows")

Reply:
Fixed

Reply:
Fixed